src/factories/ModuleFactory_v1.sol

```diff
@@ -141,12 +141,25 @@ contract ModuleFactory_v1 is
     // Public Mutating Functions
 
     /// @inheritdoc IModuleFactory_v1
-    function createModule(
+    function createAndInitModule(
         IModule_v1.Metadata memory metadata,
         IOrchestrator_v1 orchestrator,
         bytes memory configData,
         IOrchestratorFactory_v1.WorkflowConfig memory workflowConfig
     ) external returns (address) {
+        address proxy =
+            createModuleProxy(metadata, orchestrator, workflowConfig);
+
+        IModule_v1(proxy).init(orchestrator, metadata, configData);
+
+        return proxy;
+    }
+
+    function createModuleProxy(
+        IModule_v1.Metadata memory metadata,
+        IOrchestrator_v1 orchestrator,
+        IOrchestratorFactory_v1.WorkflowConfig memory workflowConfig
+    ) public returns (address) {
```

Comment:
How are these changes relevant to the issue?

Reply:
Just to resolve this: It's necessary as we need to know about the funding manager token in the staking contract (module), so a dependency injection issue resolved here. Thank you for explaining (via Huddle) @FHieser - closing this comment